dev_tools/requirements/deps/format.txt

```diff
@@ -1,2 +1,3 @@
 -r flynt.txt
 black==21.12b0
+click<=8.0.4
```

Comment:
What prompted this requirement?

Reply:
Another broad break: https://github.com/quantumlib/Cirq/runs/5725193054?check_suite_focus=true

Reply:
https://github.com/psf/black/issues/2964

Reply:
_(grumble grumble dependency hierarchy)_

SGTM. Thanks for digging into this!

Reply:
in the future you should probably factor out this fix into its own PR / commit / description so people can find it later.

Do we have an issue open for bumping to a stable version of `black`?

Reply:
PR to bump black: https://github.com/quantumlib/Cirq/pull/5157